.gitignore

```diff
@@ -11,9 +11,6 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Ignore the built binary
-cert-manager-webhook-example
```

Comment:
please ignore new name

Reply:
should be the binary you get when running `go build`

Reply:
maybe needs a `./` (or was it `/` only?) to only ignore that binary in repo root

Reply:
Indeed, `/cert-manager-webhook-anexia` works as intended.